### PR TITLE
MAINTAINERS: Add Networking: Wi-FI section

### PR DESCRIPTION
**Pull request by rlubos**
Add new "Networking: Wi-Fi" section covering Wi-Fi L2 and management concepts.

### CODE REVIEW

**Review by carlescufi — DISMISSED** [review]

**Comment by carlescufi** [comment]
Maybe this is the time to fix the spelling of "WiFi" to "Wi-Fi" everywhere:

- the drivers/wifi section
- the GitHub label

**Review by carlescufi — DISMISSED** [review]

**Comment by carlescufi** [comment]
Changed the label already from "area: WiFi" to "area: Wi-Fi"

**Review by jukkar — DISMISSED** [review]

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -1990,6 +1990,22 @@ Networking:
...
+  maintainers:
+    - jukkar
+  collaborators:
+    - rlubos
+  files:
+    - doc/connectivity/networking/api/wifi.rst
+    - include/zephyr/net/wifi*.h
+    - subsys/net/l2/wifi/
+    - samples/net/wifi/
+    - tests/net/wifi/
```
Comment (nashif) [inline MAINTAINERS.yml:2004]: you probably want to exclude those from the networking area

Comment (rlubos) [reply MAINTAINERS.yml:2009]: I've added those entries to `files-exclude:` in Networking section.

**Review by carlescufi — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**